File: src/main/java/com/spring/app/shipping/payload/ShipperData.java
package com.spring.app.shipping.payload;

public class ShipperData {
    private Long id;
    private String shipperCode;
    private String username;
    private String fullName;
    private String email;
    private String phone;
    private String address;
    private String currentAddress;
    private String createdAt;
    private String updatedAt;

    public ShipperData() {}

    public ShipperData(
            Long id,
            String shipperCode,
            String username,
            String fullName,
            String email,
            String phone,
            String address,
            String currentAddress
    ) {
        this.id = id;
        this.username = username;
        this.shipperCode = shipperCode;
        this.fullName = fullName;
        this.email = email;
        this.phone = phone;
        this.address = address;
        this.currentAddress = currentAddress;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getShipperCode() {
        return shipperCode;
    }

    public void setShipperCode(String shipperCode) {
        this.shipperCode = shipperCode;
    }

    public String getFullName() {
        return fullName;
    }

    public void setFullName(String fullName) {
        this.fullName = fullName;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getCurrentAddress() {
        return currentAddress;
    }

    public void setCurrentAddress(String currentAddress) {
        this.currentAddress = currentAddress;
    }

    public String getCreatedAt() {
        return createdAt;
    }

    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public String getUpdatedAt() {
        return updatedAt;
    }

    public void setUpdatedAt(String updatedAt) {
        this.updatedAt = updatedAt;
    }
}

File: views/src/components/models/shipping/shipper-data.js
export default class ShipperData {
    data = {
        id: null,
        shipperCode: "",
        username: "",
        fullName: "",
        email: "",
        phone: "",
        address: "",
        currentAddress: "",
        createdAt: null,
        updatedAt: null
    };

    setData (data) {
        this.data.id = data.id;
        this.data.shipperCode = data.shipperCode;
        this.data.username = data.username;
        this.data.email = data.email;
        this.data.fullName = data.fullName;
        this.data.phone = data.phone;
        this.data.address = data.address;
        this.data.currentAddress = data.currentAddress;
        this.data.createdAt = data.createdAt;
        this.data.updatedAt = data.updatedAt;
    }

    getData () {
        return this.data;
    }
}